client/components/payments-activity/payments-activity-data.tsx

```diff
@@ -10,17 +11,42 @@ import HelpOutlineIcon from 'gridicons/dist/help-outline';
  */
 import PaymentsDataTile from './payments-data-tile';
 import { ClickTooltip } from '../tooltip';
+import { usePaymentActivityData } from 'wcpay/data';
 import { getAdminUrl } from 'wcpay/utils';
+import { DateRange, PaymentsActivityData } from './types';
+
 import './style.scss';
 
+/**
+ * This will be replaces in the future with a dynamic date range picker.
+ */
+const getDateRange = (): DateRange => {
+	return {
+		// Subtract 7 days from the current date.
+		date_start: moment()
+			.subtract( 7, 'd' )
+			.format( 'YYYY-MM-DD\\THH:mm:ss' ),
+		date_end: moment().format( 'YYYY-MM-DD\\THH:mm:ss' ),
+	};
+};
+
 const PaymentsActivityData: React.FC = () => {
+	const { paymentActivityData, isLoading } = usePaymentActivityData(
+		getDateRange()
+	);
+
+	const totalPaymentsVolume = paymentActivityData?.total_payments_volume ?? 0;
+	const charges = paymentActivityData?.charges ?? 0;
+	const fees = paymentActivityData?.fees ?? 0;
+	const disputes = paymentActivityData?.disputes ?? 0;
+	const refunds = paymentActivityData?.refunds ?? 0;
+
 	return (
 		<div className="wcpay-payments-activity-data">
 			<PaymentsDataTile
 				id="wcpay-payments-activity-data__total-payments-volume"
 				label={ __( 'Total payments volume', 'woocommerce-payments' ) }
 				currencyCode="EUR"
```

Comment:
We will need to replace the primary currency in the placeholder.  
```suggestion
				currencyCode={ wcpaySettings.storeCurrency } 
```

for all the tiles.

Reply:
Oops, blunder! Fixed it :)

Reply:
Thank you 🙌